templates/expo-template-tabs/navigation/MainTabNavigator.js

```diff
@@ -1,63 +1,44 @@
 import React from 'react';
 import { Platform } from 'react-native';
-import {
-  createStackNavigator,
-  createBottomTabNavigator,
-} from 'react-navigation';
+import { createStackNavigator, createBottomTabNavigator } from 'react-navigation';
 
 import TabBarIcon from '../components/TabBarIcon';
 import HomeScreen from '../screens/HomeScreen';
 import LinksScreen from '../screens/LinksScreen';
 import SettingsScreen from '../screens/SettingsScreen';
 
-const HomeStack = createStackNavigator({
-  Home: HomeScreen,
+const components = [
+  {
+    component: HomeScreen,
+    icon: 'information-circle',
+    tabBarLabel: 'Home',
+  },
+  {
+    component: LinksScreen,
+    icon: 'link',
+    tabBarLabel: 'Links',
+  },
+  {
+    component: SettingsScreen,
+    icon: 'options',
+    tabBarLabel: 'Settings',
+  },
+];
+const stacks = {};
+components.forEach(item => {
+  let stack = createStackNavigator({
+    [item.tabBarLabel]: item.component,
+  });
+  stack.navigationOptions = {
+    tabBarLabel: item.tabBarLabel,
+    tabBarIcon: ({ focused }) => (
+      <TabBarIcon
+        focused={focused}
+        name={Platform.OS === 'ios' ? `ios-${item.icon}` : `md-${item.icon}`}
```

Comment:
Moreover, making this simpler drops support for having:
- `ios-information-circle` when tab is focused
- `ios-information-circle-outline` when tab isn't focused

which is implemented here

https://github.com/expo/expo/pull/4423/files#diff-4526cc2d38b8d3ac75e84bb5acd8a03cL24

I would think twice before adding support for `focusedIconName` since this would complicate it even more.